APILambda/user/getUser.ts

```diff
@@ -82,5 +55,6 @@ export const userAndRelationsWithId = (
       AND ur2.toUserId = :userId
       WHERE u.id = :userId;
       `,
-    { userId, fromUserId }
+    { userId, fromUserId },
+    { relations: ["themToYou", "youToThem"] }
```

Comment:
Check how this looks with more endpoints

Reply:
Maybe allow nested fields (ex. {field1: {field2: {...}})

Reply:
Instead of an extra arg, maybe chain another method in sqlexecutable like .mapnestedfields()